ChainSharp/Workflow.cs

```diff
@@ -84,7 +85,79 @@ public async Task<TReturn> Run(TInput input)
     public Workflow<TInput, TReturn> Chain<TStep, TIn, TOut>()
         where TStep : IStep<TIn, TOut>, new()
         => Chain<TStep, TIn, TOut>(new TStep());
+    
+    /// Chain<TStep>()
+    public Workflow<TInput, TReturn> Chain<TStep>() where TStep : new()
+    {
+        var stepType = typeof(TStep);
```

Comment:
Can this logic be broken up into helper functions?

Reply:
Good idea. Done.